library/psa_crypto.c

```diff
@@ -1835,7 +1835,7 @@ psa_status_t psa_import_key( const psa_key_attributes_t *attributes,
     if( driver != NULL )
     {
         const psa_drv_se_t *drv = psa_get_se_driver_methods( driver );
-        size_t bits;
+        size_t bits = PSA_MAX_KEY_BITS + 1;
```

Comment:
Why is this change needed?

Reply:
Please leave a comment in the code explaining the reason, to improve the code's readability. Others will have the same question.

Reply:
I see this change was added in commit "Parametrize key bits in import mock test", which doesn't seem right if all one intends to do is updating a test as the title would imply. If the core has to be updated together with the test change, revise the commit message. If the core has a bug, explain it in the commit message and make a change to fix the bug separate from the mock test updating.

Reply:
I've split this out into a separate commit and added an explanation.

Reply:
Please add a comment in the code about why you chose the value you did.

Reply:
Added a code comment.